web-client/src/ducks/auth/phone/actions.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  phoneAuthTrigger,
+  phoneAuthVerify,
+} from 'src/http/resources/auth/phoneAuth';
```

Comment:
I see that you split the files here and wrote the functions in separate files each representing the sub module.
However, I think that these have to be defined as functions in a file called `functions.ts` in the same directory as ducks module, that is as `src/ducks/module/functions.ts`
The reason being, these functions are required only in the action creator and we won't be interacting with the data sources directly.
We will only interact with the data sources through redux using the action creators.
If a function really is being used again in another ducks module (which I think is rare), it should be put in a `functions.ts` that is common to all the ducks module at `src/ducks/functions.ts` but still not at a http or firebase resources directory that is outside the ducks directory.
Also, I feel that it makes more sense to put the helpers related to making `HTTP` requests in `src/http/resources` rather than the actual `http` or `firebase` operations that are to be carried out.
And on this note, only helpers that aid `HTTP` requests should be put in this directory.
helpers aiding `firebase` should be in `src/firebase/resources.`
@jpmarques66

Reply:
This was corrected